### PR TITLE
Global env

### PR DESCRIPTION
**Pull request by tim775**

### CODE REVIEW

**Review by alikhajeh1 — COMMENTED** [review]
LGTM! Should we move the `INFRACOST_TERRAFORM_CLOUD*` env vars to the top-level too? Or is there a reason for repeating them twice in each workflow? maybe @aliscott knows?

**Comment by aliscott** [comment]
> Or is there a reason for repeating them twice in each workflow? maybe @aliscott knows?

I don't know of any good reason not to move them.

File: vss-extension.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "manifestVersion": 1,
   "id": "infracost-tasks",
   "name": "Infracost Tasks",
-  "version": "1.0.4",
+  "version": "1.0.5",
```
Comment (alikhajeh1) [inline vss-extension.json:5]: Something looks odd in https://github.com/infracost/infracost-azure-devops/tags, is v0.1.4 meant to be v1.0.4?

**Review by alikhajeh1 — APPROVED** [review]

File: vss-extension.json (hunk 1 above)
Comment (tim775) [reply vss-extension.json:5]: yeah must be

**Pull request merged**